docs/docs-components/MainSectionSubsection.js

```diff
@@ -94,8 +94,9 @@ function MainSectionSubsection({
               column: 4,
             }}
           >
-            {arrayChildren.map((child) => (
-              <Flex.Item flex="grow" key={child}>
+            {arrayChildren.map((child, index) => (
+              // eslint-disable-next-line react/no-array-index-key
```

Comment:
Could we use useId+index for the key so we don't have to disable this?

Reply:
@AlbertCarreras https://reactjs.org/docs/hooks-reference.html#useid

![image](https://user-images.githubusercontent.com/127199/200399719-dd2fe87f-a292-464f-8e79-5c16cc5972bc.png)